misc/dist/html/logo.svg

```diff
@@ -0,0 +1,219 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
```

Comment:
Shouldn't this be minified?

Reply:
Well, the original ( https://github.com/godotengine/godot/blob/master/logo.svg ) does not seem optimized either.